### ISSUE TITLE
SEO Audit before launch

### ISSUE THREAD

**Issue by emarchak** [CivicTechTO/women-and-color-frontend#61]

**Comment by s-kennedy** [comment]
Estimated hours: 4

**Comment by impuneet** [comment]
What is pending in the Sitemap generation now ?

**Comment by heymosef** [comment]
@impuneet If I recall correctly, we need to generate one for the new site which can continuously self-update based on new speakers and companies (ultimately be easily indexible by Google for better search optimization).

cc: @s-kennedy

**Comment by impuneet** [comment]
Url  "https://womenandcolor-frontend.herokuapp.com/" is not working

**Comment by heymosef** [comment]
The URL is beta.womenandcolor.com or women-and-color-backend.herokuapp.com

**Comment by s-kennedy** [comment]
@impuneet the app is up, but it looks like it's trying to load the js files from localhost:8080. I think it's just an incorrectly set constant. As Mohammed said, you can access the live site from beta.womenandcolor.com. 

For the site map, the main complication is that there are some routes that are handled by the React Router and some routes (for authentication) that are handled by the Django router. I think we can start with a site map for just the React routes. Try using [react-router-sitemap](https://github.com/kuflash/react-router-sitemap), it looks like the go-to package for this task.

**Comment by impuneet** [comment]
Yes as far I can understand these are some of the landing pages that  : 
**https://beta.womenandcolor.com**/#/speaker/223/LeighDoyle-LeighDoyle
**https://beta.womenandcolor.com**/#/speaker/224/Sharon-Kennedy
**https://beta.womenandcolor.com**/#/speaker/222/Stephanie-Nguyen
**https://beta.womenandcolor.com**/#/speaker/220/Nisa-Malli

We can generate the sitemap in the serverside.That's how i have done in my site in Node JS.
Having a dynamic URL : 
`baseUrl+'/'+user+'/'+userID+'/'+usernameWithSlug `

and then updating the urls after a specific period of time with cronJob.
